templates/common/config/glance-api-config.json

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
wondering why this kolla config is in common and linked to `glanceapi/config` , while the `glance-httpd-config.json` is in `glanceapi/config`?

Reply:
I think this comes from historical reasons because we went through a refactoring. long term, but out of scope for this patch, I want to only have a `glance/config` that is in "common" by default and used for all the GlanceAPI.